src/routes/todo.routes/getToDo.ts

```diff
@@ -0,0 +1,13 @@
+import { Router, Request, Response } from "express";
+import getToDos from "../../repository/todo.repository/getToDos";
+
+export default function getToDoItem(): Router {
```

Comment:
Names of the routers should be consistant. 
Best case scenario:
You have only one router per entity (in this case Todo). So `todoRouter`, which have different routes in it: `todoRouter.get("/", ... )`, `todoRouter.post("/", ...)`, ...